mlrun/runtimes/daskjob.py

```diff
@@ -407,10 +408,15 @@ def deploy(
             show_on_failure=show_on_failure,
         )
 
+    # TODO: Remove in 1.5.0
+    @deprecated(
+        version="1.3.0",
+        reason="'Dask gpus' will be removed in 1.5.0, use 'with_scheduler_limits' / 'with_worker_limits' instead",
+        category=FutureWarning,
+    )
     def gpus(self, gpus, gpu_type="nvidia.com/gpu"):
-        raise NotImplementedError(
-            "Use with_scheduler_limits/with_worker_limits to set GPU resources",
-        )
+        update_in(self.spec.scheduler_resources, ["limits", gpu_type], gpus)
```

Comment:
is it safe? mainly because gpu_type might include "." and etc. dont you need to escape it

Reply:
it is safe because the key `["limits", gpu_type]` is a list and not a string so it won't be split by `.` (it is considered as already split)